swagger/v1/component_schemas/ProviderAttributes.yml

```diff
@@ -0,0 +1,112 @@
+---
+type: object
+required:
+  - code
+  - name
+properties:
+  accredited_body?:
+    type: boolean
+    description: "Is this provider an accredited body for other provider's courses."
+    example: "true"
+  street_address_1:
+    type: string
+    description: "Building or street line one"
+    example: "Long College"
+  street_address_2:
+    type: string
+    description: "Building or street line two"
+    example: "1st floor, 86 Long Rd"
+  city:
+    type: string
+    description: "Town or city"
+    example: "London"
+  county:
+    type: string
+    description: "County"
+    example: "London"
+  changed_at:
+    type: string
+    format: date-time
+    description: |
+      Date-time timestamp of when this provider or any of it's related data changed.
+    example: "2019-06-13T10:44:31Z"
+  code:
+    type: string
+    description: |
+      Code that uniquely identifies this provider within a recruitment cycle.
+    maxLength: 3
+    minLength: 3
+    example: X99
+  created_at:
+    type: string
+    format: date-time
+    description: "Timestamp of when this provider was created."
+    example: "2019-06-13T10:44:31Z"
+  # latitude:
+  #   type: float
+  #   format:
+  #   description:
+  #   example:
+  # longitude:
+  #   type:
+  #   format:
+  #   description:
+  #   example:
```

Comment:
Anyone know the implications of exposing this data from the API? Surely it's ok for our use in Find, but beyond that?

Reply:
mmm ... Google says no?
```
You can display Geocoding API results on a Google Map, or without a map. If you want to display Geocoding API results on a map, then these results must be displayed on a Google Map. It is prohibited to use Geocoding API data on a map that is not a Google map.

```
https://developers.google.com/maps/documentation/geocoding/policies

Reply:
So how do we expose this to Find then.

Reply:
We discussed today in the tech breakdown that we can:

- remove these and either let Find to re-geocode if necessary, we may need to change how the distance search works and return the distance from the given address as part of the results
- switch to a geocode provider that doesn't have such restrictive licensing on their results